packages/agent/src/agents/supervisor/workflowController.ts

```
@@ -913,20 +915,55 @@ export class WorkflowController {
+          chunk.event === 'on_chain_start'
+        ) {
+          iteration_number++;
+        }
+        if (chunk.name === 'Branch<agent>' && chunk.event === 'on_chain_end') {
+          chunk_to_save = chunk;
+        }
 
-      if (this.timeoutId) {
-        clearTimeout(this.timeoutId);
-        this.timeoutId = null;
+        i_count++;
```

ESLint

[error] 'i_count' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)